de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/impl/PaySelectionUpdater.java

```diff
@@ -673,6 +707,17 @@ private void setTrxName(final String trxName)
 		_trxName = trxName;
 	}
 
+	/**
+	 * Invalidates Cache for current PaySelection.
+	 */
+	private void cacheInvalidationForCurrentPaySelection()
+	{
+		final IModelCacheInvalidationService modelCacheInvalidationService = Services.get(IModelCacheInvalidationService.class);
```

Comment:
declare your `modelCacheInvalidationService` service  at the top, near the other services.